README.md

```diff
@@ -71,7 +71,10 @@ Run the initialization script as follows:
 ```shell
 pyenv init
 ``` 
-... and follow the instructions to add `pyenv` to the `$PATH`.
+... and follow the instructions to add `pyenv` to the `$PATH` by editing the interactive shell configuration 
+file, `.zshrc` or `.bashrc`. If it exists, this file is a hidden file in your home directory. You can create or edit this file using a 
```

Comment:
The "Missing Semester of your CS Education" has a good article on dotfiles for the interested. Shall we include a link to that?
```suggestion
file, `.zshrc` or `.bashrc`. If it exists, this file is a hidden file ([dotfile](https://missing.csail.mit.edu/2019/dotfiles/)) in your home directory. You can create or edit this file using a 
```